lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -1020,6 +1020,10 @@ def install_puppet_agent_dev_repo_on( hosts, opts )
               # - we do not have install_32 set globally
               arch_suffix = should_install_64bit ? '64' : '86'
               release_file = "puppet-agent-x#{arch_suffix}.msi"
+            when /^osx$/
+              onhost_copy_base = File.join('/var', 'root')
+              release_path << "/apple/#{opts[:puppet_collection]}"
```

Comment:
My ruby-fu is weaksauce. I am assuming "<<" is not a leftwise shift operator, so what it do?